.github/workflows/mdbook_cd.yml

```diff
@@ -0,0 +1,29 @@
+name: Spec MDBook CD
+
+on:
+  push:
+    tags: 
+      - '*'
+jobs:
```

Comment:
Lets trigger it for any tags creation since artifacts are picked from main branch latest commit

Reply:
We have other tags we use that we don't want to trigger for however. (like EVM-v1.1.0 or such) Those would also be targeting a main commit.

Reply:
actions partially support some wild cards to filter (glob pattern but limited) ours regex will fall into [0-9]+.[0.9]+

Reply:
@wilwade  tested it it works so everytime a dated tag is created, deployment will trigger picking up last successful artifact zip from main

Reply:
@saraswatpuneet  just to double check. If I were to tag a commit that was one behind the head would it release the head or the tagged commit?

Reply:
@wilwade  good point as of now it only pics latest commit on main so it tag is behind head it will still go ahead and pick the head 

I am going to figure out how can I pick up the same commit hash as on tagged release